src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientProvider.java

```diff
@@ -47,7 +47,7 @@ public static StreamingClientProvider getStreamingClientProviderInstance() {
 
   /** ONLY FOR TESTING - to get a provider with injected properties */
   @VisibleForTesting
-  public static StreamingClientProvider injectStreamingClientProviderForTests(
+  public static StreamingClientProvider getStreamingClientProviderForTests(
```

Comment:
small personal change, because we aren't actually injecting the client provider, we are simply getting a new instance of it